compiler-rt/lib/sanitizer_common/sanitizer_symbolizer.h

```diff
@@ -136,6 +136,9 @@ class Symbolizer final {
   /// (if it wasn't already initialized).
   static Symbolizer *GetOrInit();
   static void LateInitialize();
+#if SANITIZER_WINDOWS
```

Comment:
ClearTools and no need "#if", defined and declare for all platforms, and let it be used where it's needed.